### PR TITLE
dev: Update unit tests for graphql mutations to use mocks

### LINKED ISSUE #422: Clean up: Unit tests, remove isBeta from db user init

### PR DESCRIPTION
**Pull request by abbyoung**
## Description 

Now that we are not using `@client` in our queries, we can update our unit tests to take advantage of Apollo's mock capabilities in unit tests.

* Updates unit tests using graphql operations
* Moves cms data mocks and getCollections mock into `__fixtures__` directory
* Removes `isBeta` from portal db since we are not using it

Fixes #422 

## Review Notes

To Do
- [x] Replace mocked mutations in sites-and-apps.test.tsx 
- [x] sync with suz on fixtures naming/organization
- [x] fix add to new collection from s&a alpha page

### CODE REVIEW

**Review by suzubara — COMMENTED** [review]
I started a `__fixtures__` directory also in https://github.com/USSF-ORBIT/ussf-portal-client/pull/408 ! It might be useful to organize by type of fixture, like data vs graphql operations..

File: .github/workflows/node-ci.yml [hunk 1]
```diff
@@ -11,12 +11,12 @@ jobs:
...
     steps:
       - name: Check out repository
-        uses: actions/checkout@5a4ac9002d0be2fb38bd78e4b4dbde5606d7042f # renovate: tag=v2
+        uses: actions/checkout@ec3a7ce113134d7a93b817d10a8272cb61118579 # renovate: tag=v2
 
       - name: Set up node
         uses: actions/setup-node@04c56d2f954f1e4c69436aa54cfef261a018f458 # renovate: tag=v2
         with:
-          node-version: '14.18.1'
+          node-version: '14.18.2'
```
Comment (abbyoung) [inline .github/workflows/node-ci.yml:19]: unclear how/why these changes are in this branch and not main, and how I got them. will review tomorrow 😩

Comment (abbyoung) [reply .github/workflows/node-ci.yml:19]: it's showing a number of files in changed that are just in main... maybe it will resolve itself tomorrow.

Comment (abbyoung) [reply .github/workflows/node-ci.yml]: ⭐ resolved ⭐

**Review by suzubara — DISMISSED** [review]

**Review by suzubara — APPROVED** [review]
